src/libs/Middleware/SaveResponseInOnyx.ts

```diff
@@ -1,20 +1,15 @@
-import _ from 'underscore';
 import CONST from '../../CONST';
 import ONYXKEYS from '../../ONYXKEYS';
 import * as MemoryOnlyKeys from '../actions/MemoryOnlyKeys/MemoryOnlyKeys';
 import * as OnyxUpdates from '../actions/OnyxUpdates';
+import Middleware from './types';
 
 // If we're executing any of these requests, we don't need to trigger our OnyxUpdates flow to update the current data even if our current value is out of
 // date because all these requests are updating the app to the most current state.
 const requestsToIgnoreLastUpdateID = ['OpenApp', 'ReconnectApp', 'GetMissingOnyxMessages'];
 
-/**
- * @param {Promise} requestResponse
- * @param {Object} request
- * @returns {Promise}
- */
-function SaveResponseInOnyx(requestResponse, request) {
-    return requestResponse.then((response) => {
+const SaveResponseInOnyx: Middleware = (requestResponse, request) =>
```

Comment:
Add types to these

Reply:
Same as https://github.com/Expensify/App/pull/28031#discussion_r1345580714